pkg/rke/k8s_rke_system_images.go

```diff
@@ -11588,6 +11628,86 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 		},
+		// Enabled in Rancher v2.11.0
+		"v1.31.6-rancher1-1": {
+			Etcd:                      "rancher/mirrored-coreos-etcd:v3.5.15",
+			Kubernetes:                "rancher/hyperkube:v1.31.6-rancher1",
+			Alpine:                    "rancher/rke-tools:v0.1.109",
+			NginxProxy:                "rancher/rke-tools:v0.1.109",
+			CertDownloader:            "rancher/rke-tools:v0.1.109",
+			KubernetesServicesSidecar: "rancher/rke-tools:v0.1.109",
+			KubeDNS:                   "rancher/mirrored-k8s-dns-kube-dns:1.23.0",
+			DNSmasq:                   "rancher/mirrored-k8s-dns-dnsmasq-nanny:1.23.0",
+			KubeDNSSidecar:            "rancher/mirrored-k8s-dns-sidecar:1.23.0",
+			KubeDNSAutoscaler:         "rancher/mirrored-cluster-proportional-autoscaler:v1.8.9",
+			Flannel:                   "rancher/mirrored-flannel-flannel:v0.25.1",
+			FlannelCNI:                "rancher/flannel-cni:v1.4.1-rancher1",
+			CalicoNode:                "rancher/mirrored-calico-node:v3.29.0",
+			CalicoCNI:                 "rancher/calico-cni:v3.29.0-rancher1",
+			CalicoControllers:         "rancher/mirrored-calico-kube-controllers:v3.29.0",
+			CalicoCtl:                 "rancher/mirrored-calico-ctl:v3.29.0",
+			CalicoFlexVol:             "rancher/mirrored-calico-pod2daemon-flexvol:v3.29.0",
+			CanalNode:                 "rancher/mirrored-calico-node:v3.29.0",
+			CanalCNI:                  "rancher/calico-cni:v3.29.0-rancher1",
+			CanalControllers:          "rancher/mirrored-calico-kube-controllers:v3.29.0",
+			CanalFlannel:              "rancher/mirrored-flannel-flannel:v0.25.1",
+			CanalFlexVol:              "rancher/mirrored-calico-pod2daemon-flexvol:v3.29.0",
+			AciCniDeployContainer:     "noiro/cnideploy:6.1.1.2.81c2369",
+			AciHostContainer:          "noiro/aci-containers-host:6.1.1.2.81c2369",
+			AciOpflexContainer:        "noiro/opflex:6.1.1.2.81c2369",
+			AciMcastContainer:         "noiro/opflex:6.1.1.2.81c2369",
+			AciOpenvSwitchContainer:   "noiro/openvswitch:6.1.1.2.81c2369",
+			AciControllerContainer:    "noiro/aci-containers-controller:6.1.1.2.81c2369",
+			PodInfraContainer:         "rancher/mirrored-pause:3.7",
+			Ingress:                   "rancher/nginx-ingress-controller:nginx-1.12.0-rancher1",
+			IngressBackend:            "rancher/mirrored-nginx-ingress-controller-defaultbackend:1.5-rancher2",
+			IngressWebhook:            "rancher/mirrored-ingress-nginx-kube-webhook-certgen:v1.4.1",
+			MetricsServer:             "rancher/mirrored-metrics-server:v0.7.1",
+			CoreDNS:                   "rancher/mirrored-coredns-coredns:1.11.3",
+			CoreDNSAutoscaler:         "rancher/mirrored-cluster-proportional-autoscaler:v1.8.9",
+			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.7",
+			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.23.0",
+		},
+		// Enabled in Rancher v2.11.0
+		"v1.32.2-rancher1-1": {
+			Etcd:                      "rancher/mirrored-coreos-etcd:v3.5.15",
```

Comment:
This should be v3.5.16, https://github.com/kubernetes/kubernetes/blob/67a30c0adcf52bd3f56ff0893ce19966be12991f/build/dependencies.yaml#L67

Reply:
updated in images mirror already - https://github.com/rancher/image-mirror/blob/517a782486143292f693a9188e2e8aff1b4ef8c2/images-list#L1689

Reply:
updated @kinarashah